README.md

```diff
@@ -166,14 +157,6 @@ const [seconds, nanos] = date.toTuple();
 
 This library follows [Semantic Versioning](http://semver.org/).
 
-This library is considered to be in **alpha**. This means it is still a
```

Comment:
Should we still say whatever it is?

Reply:
Personally I think if the version is not 1.x it's implied, buuut given it's part of the googleapis org I think we'll want a Googler to chime in here. @JustinBeckwith have any opinions?

Reply:
This is fine honestly.